data/korath outfits.txt

```diff
@@ -137,6 +137,18 @@ outfit "Systems Core (Large)"
 	"energy capacity" 5200
 	description `This massive bank of computers, shield generators, and repair stations holds everything needed to keep the very largest of Korath ships in working condition.`
 
+
+outfit "Fuel Processor"
+        category "Systems"
+        cost 400000
+        thumbnail "outfit/unknown"
+        "mass" 14
+        "outfit space" -14
+        "fuel capacity" 200
+        "ramscoop" 2
+        description "Since the dawn of spaceflight the Korath have probably spent more time on military campaigns than at home, so they needed their fleets to have an ample range. This is a miniaturized Korath fuel refinery for use in expeditionary capital ships, complete with a small ramscoop and a large storage tank."
```

Comment:
Perhaps a different description:
`The Fuel Processor was produced during the latter half of the Korath Civil War, during the time when Korath factions attempted to subsidize their war effort by raiding the Hai for supplies. Korath Raiders were often destroyed after they had run out of fuel and were unable to escape from the Hai military fleets, prompting the creation of this hybrid fuel pod and ramscoop device.`

And maybe add `Fuel Processors are now the main source of fuel for the remaining Korath in the galaxy.` at the end as an explanation for the modern use.

Reply:
`during the latter half` sounds a bit awkward to me, maybe just using `during the second half` would do, or try something similar?

Reply:
I changed the description, heavily using what you've suggested. Hopefully I didn't make too many mistakes while at it.

Reply:
why use the word "portable" at all? portable is usually something that's easily carried or moved, and I doubt the outfit would qualify
and I'd drop the word "device" in your version, because `refinery with integrated ramscoop device` sounds awkward:
`hybrid fuel pod and ramscoop` is a mixed device between those 2 components, while in `refinery with integrated ramscoop` it's not a mix between those 2; the ramscoop is integrated in the refinery, meaning only the refinery is the device
`refinery device with integrated ramscoop` would technically work but sounds awkward (and I'm not sure if that's even proper grammar)

although if ya want internet facts & links to grammar rules, I'm not your guy